pkg/raft/opts.go

```diff
@@ -46,13 +68,19 @@ var (
 )
 
 func init() {
-	flag.StringVar(&opts.nodeUrlStr, "nexusNodeUrl", "", "Url for the Nexus service to be started on this node (format: http://<local_node>:<port_num>)")
-	flag.StringVar(&opts.logDir, "nexusLogDir", "/tmp/logs", "Dir for storing RAFT logs")
-	flag.StringVar(&opts.snapDir, "nexusSnapDir", "/tmp/snap", "Dir for storing RAFT snapshots")
-	flag.StringVar(&opts.clusterUrl, "nexusClusterUrl", "", "Comma separated list of Nexus URLs of other nodes in the cluster")
-	flag.Int64Var(&replTimeoutInSecs, "nexusReplTimeout", 5, "Replication timeout in seconds")
-	flag.BoolVar(&opts.leaseBasedReads, "nexusLeaseBasedReads", true, "Perform reads using RAFT leader leases")
-	flag.StringVar(&opts.statsdAddr, "nexusStatsDAddr", "", "StatsD server address (host:port) for relaying various metrics")
+
+	flag.StringVar(&opts.nodeUrlStr, "nexus-node-url", "", "Url for the Nexus service to be started on this node (format: http://<local_node>:<port_num>)")
+	flag.StringVar(&opts.logDir, "nexus-log-dir", "/tmp/logs", "Dir for storing RAFT logs")
+	flag.StringVar(&opts.snapDir, "nexus-snap-dir", "/tmp/snap", "Dir for storing RAFT snapshots")
+	flag.StringVar(&opts.clusterUrl, "nexus-cluster-url", "", "Comma separated list of Nexus URLs of other nodes in the cluster")
+	flag.Int64Var(&replTimeoutInSecs, "nexus-repl-timeout", defaultRaftReplTimeout, "Replication timeout in seconds")
+	flag.BoolVar(&opts.leaseBasedReads, "nexus-lease-based-reads", true, "Perform reads using RAFT leader leases")
+	flag.StringVar(&opts.statsdAddr, "nexus-statsd-addr", "", "StatsD server address (host:port) for relaying various metrics")
+
+	flag.IntVar(&opts.maxSnapFiles, "max-snapshots", defaultMaxSNAP, "Maximum number of snapshot files to retain (0 is unlimited)")
```

Comment:
Please prefix the new options with `nexus-`. This is used in DKV for filtering Nexus options. Refer [here](https://github.com/flipkart-incubator/dkv/blob/185739a2791afc3665f755fbbb2b5110ef9307fa/cmd/dkvsrv/main.go#L107).